src/wazuh_db/wdb_agents.c

```diff
@@ -88,24 +88,30 @@ cJSON* wdb_agents_insert_vuln_cves(wdb_t *wdb,
                                    const char* reference,
                                    const char* type,
                                    const char* status,
-                                   bool check_pkg_existence) {
+                                   bool check_pkg_existence,
+                                   const char* severity,
+                                   double cvss2_score,
+                                   double cvss3_score) {
 
+    bool insert = FALSE;
     cJSON* result = cJSON_CreateObject();
     if (!result) {
         return NULL;
     }
 
     if (wdb_agents_find_cve(wdb, cve, reference)) {
         cJSON_AddStringToObject(result, "action", "UPDATE");
+        insert = FALSE;
```

Comment:
Is this necessary, since **insert** is already FALSE?

Reply:
You are right, it's not necessary